tests/galsim_test_helpers.py

```diff
@@ -361,7 +361,8 @@ def all_obj_diff(objs):
         for j, objj in enumerate(objs):
             if i == j:
                 continue
-            assert obji != objj, "Found equivalent objects {} == {}".format(obji, objj)
+            assert obji != objj, ("Found equivalent objects {} == {} at indices {} and {}"
+                                  .format(obji, objj, i, j))
```

Comment:
At least when we started working on GalSim python 2.6 was still the default python version packaged on some linux distributions (maybe it was red hat?).  I think that was one reason we still wanted to support it.  But it's been a few years by now and it's hard to imagine that that hasn't changed.  (Or, in the case of the cluster I'm on, 2.6 might still be the default, but the sys admins installed 2.7 ages ago because everyone wanted it.)